webbpsf/opds.py

```diff
@@ -1248,8 +1248,7 @@ def _apply_hexikes_to_seg(self, segment, hexike_coeffs, debug=False):
 
         apmask = np.ones_like(Xc) # by construction, we're only evaluating this for the good pixels
 
-        hexikes = zernike.hexike_basis_wss(x=Xc, y=Yc, nterms=len(hexike_coeffs),
-            aperture=apmask)
+        hexikes = zernike.hexike_basis_wss(x=Xc, y=Yc, nterms=len(hexike_coeffs))
```

Comment:
Hmmm, I remember adding the aperture mask parameter during the JSC tests as part of modeling the JSC unstacked image array configuration. There was a reason it was there. But something does seem to have gotten out of sync between this and poppy. I'll have to look into this on Monday. Either I'll merge this as is or figure out why the aperture mask parameter is missing in the poppy function definition. Thanks.